src/uws_client.c

```diff
@@ -1116,8 +1122,12 @@ static void on_underlying_io_bytes_received(void* context, const unsigned char*
             case UWS_STATE_CLOSING_WAITING_FOR_CLOSE:
             {
                 /* Codes_SRS_UWS_CLIENT_01_385: [ If the state of the uws instance is OPEN, the received bytes shall be used for decoding WebSocket frames. ]*/
-                unsigned char* new_received_bytes = (unsigned char*)realloc(uws_client->stream_buffer, uws_client->stream_buffer_count + size + 1);
-                if (new_received_bytes == NULL)
+                unsigned char* new_received_bytes;
+
+                //size_t realloc_size = uws_client->stream_buffer_count + size + 1; **using safe int**
```

Comment:
Should remove?

Reply:
No, I want it there to show the intended math using safe macros